src/promptflow-devkit/promptflow/_sdk/operations/_flow_operations.py

```diff
@@ -1018,7 +1018,6 @@ def _merge_signature(extracted, signature_overrides):
     @staticmethod
     def _infer_signature(entry: Union[Callable, FlexFlow, Flow, Prompty], include_primitive_output: bool = False):
```

Comment:
@elliotzh , does this executes in current process or in new process?

Reply:
offline synced.